plugins/capistrano/capistrano.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+# Added `shipit` because `cap` is a reserved word. `cap` completion doesn't work.
```

Comment:
Technical niggle: `cap` is not a reserved word; it is a shell builtin, supplied by the `zsh/cap` module. Won't make a difference to most users, but they interact differently with shell customization code.